evansmusomi/burger-builder
# Adds Unit Tests

*What this does*
- Adds enzyme for unit testing
- Adds <NavigationItems> component tests
- Adds <BurgerBuilder> container tests
- Adds <Auth> reducer tests

*How to verify*
- A developer should be able to run the test suite and see the 6 defined tests passing
